modules/capi-yaml-machine-template-vsphere.adoc

```diff
@@ -51,8 +51,8 @@ The following values are valid:
 +
 When using the `linkedClone` type, the disk size matches the clone source instead of using the `diskGiB` value.
 For more information, see the {vmw-short} documentation about VM clone types.
-<7> Specify the vCenter Datacenter to deploy the compute machine set on.
-<8> Specify the vCenter Datastore to deploy the compute machine set on.
+<7> Specify the vCenter data center to deploy the compute machine set on.
+<8> Specify the vCenter datastore to deploy the compute machine set on.
```

Comment:
ISG uses "data store", but our doc seem to prefer "datastore". Shrug.

```suggestion
<8> Specify the vCenter data store to deploy the compute machine set on.
```

Reply:
this is specifically for VMware, and they have "[datastore](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-D5AB2BAD-C69A-4B8D-B468-25D86B8D39CE.html)" as a single word. I agree elsewhere/generically it should be two words though